geode-web-api/src/integrationTest/java/org/apache/geode/rest/internal/web/controllers/RestAccessControllerTest.java

```diff
@@ -740,12 +889,76 @@ public void deleteMultipleKeys() throws Exception {
 
   @Test
   @WithMockUser
-  public void deleteMultipleKeysWithSlashes() throws Exception {
-    putAllWithSlashes();
-    mockMvc.perform(delete("/v1/customers/1" + SLASH + ",2" + SLASH + ",3"
-        + SLASH + ",4" + SLASH + ",5" + SLASH)
-            .with(POST_PROCESSOR))
+  public void deleteMultipleKeysWithQueryParam() throws Exception {
+    putAll();
+    assertThat(customerRegion.size()).isEqualTo(60);
```

Comment:
I think you can use AssertJ's `Map` assertions on regions:
```
assertThat(customerRegion)
    .hasSize(60)
    .containsKeys("2", "3", "4", "5");
```
This will make the assertion failures more helpful.

Reply:
I changed over to hasSize.